frontend/src/components/Home.js

```diff
@@ -31,22 +25,31 @@ export default class Home extends Component {
             });
             this.authenticateSpotify();
         });
-    }
+    }*/
 
     //Ask if current user is authenticated 
     authenticateSpotify() {
         fetch('/spotify/is-authenticated')
             .then((response) => response.json())
             .then((data) => {
-                this.setState({spotifyAuthenticated: data.status })
+                this.setState({ spotifyAuthenticated: data.status })
                 if (!data.status) {
                     fetch('/spotify/get-auth-url')
-                    .then((response) => response.json())
-                    .then((data) => {
-                        window.location.replace(data.url);
-                    });
+                        .then((response) => response.json())
+                        .then((data) => {
+                            window.location.replace(data.url);
+                        });
                 }
-        });
+            });
+    }
+
+    //User logout
+    logout() {
```

Comment:
Logout of Spotify or LOST? Maybe more specific function name? Doesn't really matter.

Reply:
good idea, thank you!